config/params/minimal_config.go

```diff
@@ -112,7 +112,7 @@ func MinimalSpecConfig() *BeaconChainConfig {
 	minimalConfig.MaxDepositRequestsPerPayload = 4
 	minimalConfig.PendingPartialWithdrawalsLimit = 64
 	minimalConfig.MaxPendingPartialsPerWithdrawalsSweep = 2
-	minimalConfig.PendingDepositLimit = 134217728
```

Comment:
This is the same value as mainnet? This isn't an override so you could just delete this line, if you want.